### ISSUE TITLE
can use_gound_truth make result better？ why use_gound_truth?

### ISSUE THREAD

**Issue by alphanlp** [rohithreddy024/Text-Summarizer-Pytorch#17]

**Comment by rohithreddy024** [comment]
In [Deep Reinforced Molel paper](url), they mentioned ,

> For ML training, we use the teacher forcing algorithm with the only difference that at each decoding
step, we choose with a 25% probability the previously generated token instead of the ground-truth
token as the decoder input token yt−1, which reduces exposure bias. 

use_ground_truth indicates whether to use ground truth label instead of previously predicted token  at each decoding step. So yes, this mechanism reduces exposure bias and thus make results better.
